dataGenerator.js

```diff
@@ -55,45 +78,209 @@ while (numberOfReviews <= 1000) {
     countdown--;
   }
 
+  // populate products array with current product + characteristics
   products[product_id] = {
     product_id: product_id + 1,
     characteristics: fourRandomCharacteristics
   };
 
-  let randomReviewAmount = random(0, 8);
-
+  let randomReviewAmount = random(0, 10);
   for (var i = 0; i <= randomReviewAmount; i++) {
-    // let product_id = products[i].product_id;
+    // generate review
     let date = faker.date.between('2020-06-09', '2020-09-09');
-    let review = `${numberOfReviews + 1},${product_id + 1},${random(1, 6)},${date.toString().slice(0, 24)},${faker.random.words()},${faker.lorem.paragraph()},${(Math.random() >= .3)},${false},${faker.name.findName()},${faker.internet.email()},${"none"},${random(0, 30)}`
+    if (numberOfReviews / numberToWrite < .1) {
+      reviews1 += `${numberOfReviews + 1},${product_id + 1},${random(1, 6)},${date.toString().slice(0, 24)},${faker.random.words()},${faker.lorem.paragraph()},${(Math.random() >= .3)},${false},${faker.name.findName()},${faker.internet.email()},${"none"},${random(0, 30)}\n`
+    } else if (numberOfReviews / numberToWrite < .2) {
```

Comment:
numberOfReviews / numberToWrite is said often enough, should it have variable? 
let reviewRatio = numberOfReviews / numberToWrite
If you choose to make this into a variable think about where you should define the variable so that it changes when the numOfRevs changes.